packages/core/useWebLocks/index.test.ts

```diff
@@ -0,0 +1,8 @@
+import { describe, expect, it } from 'vitest'
+import { useWebLocks } from '.'
+
+describe('useWebLocks', () => {
+  it('should be defined', () => {
+    expect(useWebLocks).toBeDefined()
+  })
+})
```

Comment:
Maybe we can test some things here.

We can also use browser tests to use a real headless browser.

Reply:
Added tests. But unfortunately, my Vitest knowledge is insufficient – maybe you can tell me how to catch the [Unhandled Rejection](https://github.com/vueuse/vueuse/blob/78aa3c47127877b9b5dbf6fde73942d99fa5bbdc/packages/core/useWebLocks/index.browser.test.ts#L57-L58) error?

Reply:
not that easy. I think we dont need to test the `tryOnScopeDispose`.

Reply:
I found [a way to test/catch it](https://github.com/vitest-dev/vitest/pull/6016)